### DISCUSSION TITLE
Reverify your academic affiliation issue

### DISCUSSION THREAD

**Discussion by eyupkaganeyupoglu** [community/community#54187] [GitHub Education]
Continue is not clickable. I couldn't understand why.
![Ekran Alıntısı](https://user-images.githubusercontent.com/72672133/235511990-9e8807c9-1d5e-45e8-b2f7-b98676ef96b4.PNG)

**Comment by AliAmmarNaseem** [comment]
In my case it is clickable but getting this error "We require applicants to use a school-issued email address to apply."

**Comment by ElliotQi** [comment]
I meet this problem too. Did you solve it?

**Comment by samus-aran** [comment]
Hi @hirelaxstudio & @ElliotQi are you both users that need to re-verify? What does the status say on the [application status page?](https://education.github.com/discount_requests/student_application)

**Reply by eyupkaganeyupoglu** [reply]
Yes.
![Ekran Alıntısı3](https://user-images.githubusercontent.com/72672133/235730403-e8622cf2-a124-4c62-830c-136701f1224f.PNG)
![Ekran Alıntısı2](https://user-images.githubusercontent.com/72672133/235682029-3487906c-5810-4d0c-a313-a14a325a0da4.PNG)
![Ekran Alıntısı](https://user-images.githubusercontent.com/72672133/235730564-64a4c5e5-011f-4341-b0bb-ae4c8d888ded.PNG)

**Reply by ElliotQi** [reply]
same as his @hirelaxstudio
